Template/layout/layout.php

```diff
@@ -20,6 +20,32 @@
         <?php endif ?>
         <?= $this->asset->customCss() ?>
 
+        <!-- Customizer personalized style -->
+        <?php global $customizer; ?>
+        <style>
+            <?php if($customizer['background_color']) : ?>
+                body {
+                    background-color: <?= $customizer['background_color'] ?>!important;
```

Comment:
If we go this route:

you can directly pull the value from the DB here, bypassing the global var. `$this->task->configModel->get(key, default value)`

The other option, would be to write all this to a css file, if we do not use the parse method from an existing theme, we can still build a css file using these values. This would also allow users to save different custom themes, whereas this method will not.